pom.xml

```diff
@@ -23,6 +23,7 @@
       <module>commons-test</module>
       <module>core</module>
       <module>extended-statistics</module>
+      <module>distributed-counter</module>
```

Comment:
I know we've used the term 'distributed counters', but wouldn't 'clustered counters' be more fitting?